west.yml

```diff
@@ -67,7 +67,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: bd2e2d8dcdc19defdb1bf6e07fb94f62bc99c925
+      revision: bcd7565ffa390d5774dc2fbe71a002faa9a7d082
```

Comment:
Commit message of https://github.com/zephyrproject-rtos/zephyr/pull/41920/commits/3342d5027be4556414c67edd7df791514b300401 says:
> This change updates startup code to setup MMU mappings 

but it seems that commit also updates hal_espressif to a newer version.

Please add a sentence in the commit message what kind of changes is brought in with sha update / reason for updating the manifest.